ffw/BasicCommunicationRPC.js

```diff
@@ -728,6 +731,55 @@ FFW.BasicCommunication = FFW.RPCObserver
         };
         this.client.send(JSONMessage);
       },
+
+      /**
+      * Response with current system UTC time
+      *
+      * @param {Number}
+      *            id
+      * @param {String}
+      *            method
+      */
+      GetSystemTime : function(id, method) {
+        Em.Logger.log('FFW.' + method + ' Response');
+
+        var current_time = new Date;
+        var current_tz_hour = Math.floor(-current_time.getTimezoneOffset() / 60);
+        var current_tz_minute =
+            (-current_time.getTimezoneOffset()) - (current_tz_hour * 60);
+        if (current_tz_minute < 0) {
+          current_tz_minute += 60;
+          if (current_tz_hour == 14)
+            current_tz_hour = -12;
+          else
+            current_tz_hour++;
+        }
```

Comment:
@Kozoriz, yes, it's good idea. But there were some clarifications for this function and there is no need to calculate this fields anymore (see [APPLINK-33109](https://adc.luxoft.com/jira/browse/APPLINK-33109)). I've changed it in [43c9ff7](https://github.com/CustomSDL/sdl_hmi/pull/4/commits/43c9ff7ae8ada9211f2ed5ef1b79da294f8478ed)